Error in filename in index.js

This error pops when loading `index.js` on a Linux environment.
There is no issue on MacOSX because it does not care about files names being caps locked or not.

```
Error: ./node_modules/@internet/raf/lib/index.js
  Module not found: Error: Can't resolve './FpsLimiter.js' in '/usr/src/app/node  _modules/@internet/raf/lib'
  resolve './FpsLimiter.js' in '/usr/src/app/node_modules/@internet/raf/lib'
```

--- a/lib/index.js
+++ b/lib/index.js
@@ -1,5 +1,5 @@
 import RafTimer from './RafTimer.js'
-import fpsLimiter from './FpsLimiter.js'
+import fpsLimiter from './fpsLimiter.js'
 import raf from './raf.js'
 
 export { RafTimer, fpsLimiter, raf }

